app-frontend/src/app/pages/projects/edit/annotate/annotate.controller.js

```diff
@@ -23,6 +25,8 @@ export default class AnnotateController {
         this.isUpload = true;
         this.bindUploadEvent();
 
+        this.clickedId = '';
+
         this.$scope.$on('$destroy', this.$onDestroy.bind(this));
     }
```

Comment:
I'd move the `$onDestroy` function under `onInit` since the two functions are functionally related (build + teardown)